test/conftest.py

```diff
@@ -33,6 +38,8 @@ def pytest_addoption(parser):
     parser.addoption('--tmpdir', action='store', default='testlog', help='''Path to temporary test data and log files. The data is
             further segregated per build mode. Default: ./testlog.''', )
     parser.addoption('--run_id', action='store', default=None, help='Run id for the test run')
+    parser.addoption('--byte-limit', action="store", default=randint(0, 2000), type=int,
+                     help="Specific byte limit for failure injection (random by default)")
```

Comment:
Again, I don't know what this means, or why any number between 0 and 2000 (including 0) is fine. Who needs this option? Where is it used?

Reply:
This code is originally from [PR](https://github.com/scylladb/scylladb/pull/22030/files#diff-3665d65394f4f58a56a256ad6dd8621c68118d90fe56a19387e251c19cec2d2e). I'm just moving to another place.